### ISSUE TITLE
Feature request: Option for generating pipewire filter-chain configs as output

### ISSUE THREAD

**Issue by DependentAppearance0** [jaakkopasanen/AutoEq#822]
**Is your feature request related to a problem? Please describe.**
On linux using the pipewire audio system there are multiple ways of applying equalization. While solutions like easyeffects work the easyeffects application is quite heavy for anyone just wanting equalization. And to this pipewire has its own solution which is its filter-chain feature. Using the filter chain a user can define a custom filters on their audio sources, which includes equalization. However at the current moment AutoEq does not support generating these config files which means users have to manually type the values in.

**Describe the solution you'd like**
It would be very nice if the AutoEq application could directly generate these configuration files for users. The config format is fairly simple but copying values manually is not fun.

An example config file can be found at
https://gitlab.freedesktop.org/pipewire/pipewire/-/blob/master/src/daemon/filter-chain/sink-eq6.conf
And additional information about supported features are documented at
https://docs.pipewire.org/page_module_filter_chain.html

The example config shows a parametric equalizer using biquad filters but other filters are also supported in the chain.

**Describe alternatives you've considered**
Currently a pipewire config file can be created from AutoEq data by manually filling out the filter values from the application but it would obviously be nicer to just download a configuration file and be done. Alternatively other software such as easyeffects could be used but using a filter-chain eliminates the need for extra software on any linux system using pipewire.

**Comment by eduardoeae** [comment]
Hello. Have you tested this? I tried making this config manually one time, but the CPU usage was way higher than just using easyeffects, so I just use easyeffects nowadays.

**Comment by DependentAppearance0** [comment]
Yes. I use a filter-chain for both eq and mic noise suppression daily and it works far more reliably than easyeffects. I haven't noticed any cpu usage issues either.

Overall I stopped using easyeffects as I found its configuration editor to be buggy (did not allow me to properly configure the rnnoise plugin) and overall its reliability was not great as it or pipewire crashing would take all sound down completely until I manually restarted the application. For example putting the system to sleep would break audio using easyeffects as pipewire was suspended during the sleep and easyeffects didnt recover.

**Comment by EmersonBiggons** [comment]
1+ This would be a fantastic addition. Especially for arch users with an anti bloat mindset, such and such.
